README.rst

```diff
@@ -38,6 +38,72 @@ HTML gallery of examples from any set of Python scripts.
    :width: 80%
    :alt: A demo of a gallery generated by Sphinx-Gallery
 
+Quickstart
+==========
+
+Sphinx-Gallery can be used to generate an example gallery for a library, as well as a stand-alone web page showcasing examples of a particular Python package, module, or class.
+
+Let's get started with a simple example or checkout the `documentation <sphinx-gallery.github.io>`_ for introductions on how to use it and more...
+
+Install via ``pip``
+-------------------
+
+.. installation-begin-content
+
+You can do a direct install via ``pip`` by using:
+
+.. code-block:: bash
+
+    $ pip install sphinx-gallery "sphinx>=4.0" pillow
+
+.. warning::
+    Sphinx-Gallery will not manage its dependencies when installing, thus
+    you are required to install them manually. Our minimal dependencies
+    are **Sphinx >= 4** and Pillow, which we use for scaling images.
+
+.. note::
+    Sphinx-Gallery has also support for scraping images from packages like
+    Matplotlib, Seaborn, and PyVista. We recommend installing system ``optipng``
+    binaries to reduce the file sizes of the generated PNG files.
+
+.. installation-end-content
+
+Add examples to your docs
+-------------------------
+
+Let's assume simple scenario, you have a Python package with a directory structure like this:
+
+.. code-block::
+    .
+    ├── doc
+    │   ├── conf.py
+    │   ├── index.rst
+    |   ├── make.bat
+    │   └── Makefile
+    ├── my_python_module
+    │   ├── __init__.py
+    │   └── mod.py
+    └── examples
+        ├── plot_example.py
+        └── README.txt (or .rst)
+
+Then you just add the following to your ``doc/conf.py``:
+
+.. code-block:: python
+
+    extensions = [
+        ...
+        'sphinx_gallery.gen_gallery',
+    ]
+
+    # path to the examples scripts
+    sphinx_gallery_conf = {
+        'examples_dirs': '../examples',   # path to your example scripts
+        'gallery_dirs': 'gallery',  # path to where to save gallery generated output
+    }
+
+Finally just compile your docs as usual. You will see a new section "gallery" in your docs with the examples.
```

Comment:
I think you need to add the gallery index (e.g., `examples/index.rst`) file to the main `doc/index.rst` file for it to appear in the docs ?

Reply:
yes that is good addition